index.js

```diff
@@ -0,0 +1 @@
+console.log("Welcome to my App");
```

Comment:
Hey @ellie

I've gone with this message - wasn't too sure whether to go with App or Application?